jdi-light-bootstrap/src/main/java/com/epam/jdi/light/ui/bootstrap/interfaces/IsNavbarBrand.java

```diff
@@ -0,0 +1,5 @@
+package com.epam.jdi.light.ui.bootstrap.interfaces;
+
+public interface IsNavbarBrand {
```

Comment:
NavbarBrand is any custom element. You shouldn't create such element in bootstrap package, because it doesn't have any specific and unique properties or methods.

Reply:
It can be collected in JList now. For common elements you can not do it.